nav2_dwb_controller/dwb_critics/src/oscillation.cpp

```diff
@@ -104,7 +104,7 @@ void OscillationCritic::onInit()
    * If min_trans_vel is set in the namespace, as it used to be used for trajectory generation, complain then use that.
    * Otherwise, set x_only_threshold_ to 0.05
    */
-  nh_->get_parameter_or("x_only_threshold", x_only_threshold_, 0.05);
+  nh_->get_parameter_or(name_+ "/x_only_threshold", x_only_threshold_, 0.05);
```

Comment:
Should this be:

`name_ + ".x_only_threshold`

to match the other parameters?